README.md

```diff
@@ -0,0 +1,71 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+>This project based on the linter configuration. setup project built with html and css
+
+![Screenshot](https://user-images.githubusercontent.com/61976324/143077519-7ec39aed-d790-4a2e-9e17-0a684381cc73.png)
+
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages: html and css
+- Frameworks: NO
+- Technologies used: VScode, chrome, and github
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```

Comment:
- [ ] You have done an excellent job with your project but I would suggest removing the sections that are not in use to make your README look even more professional :raised_hands:
![image](https://user-images.githubusercontent.com/81522435/143078550-ebc4bbf0-0fd5-4da6-b10e-081e3613817a.png)